client/my-sites/domains/domain-management/edit/card/renew-button/style.scss

```diff
@@ -0,0 +1,5 @@
+.renew-button {
+	&.is-loading strong {
+		animation: loading-fade 1.6s ease-in-out infinite;
```

Comment:
I like this small touch :) I wonder, though, if for the sake of consistency we should use a `Button` with `busy` property instead? https://wpcalypso.wordpress.com/devdocs/design/button (unfortunately, the devdocs for Button seems to be broken now, see https://wpcalypso.wordpress.com/devdocs/design/button-group instead)